stacks/syrah/syrah.yaml

```diff
@@ -765,7 +765,7 @@ mpi_packages:
     - openfoam-org +metis ^scotch+mpi
     - phylobayesmpi
     - scotch +mpi
-    - wrf ~pnetcdf build_type=dm+sm ^netcdf-c +mpi ^netcdf-fortran
+    - wrf ~pnetcdf build_type=dm+sm ^netcdf-c +mpi ^netcdf-fortran target=<target>
```

Comment:
I would have added the target variant before the dependencies.

Reply:
why ? when you ask a `spec` to spack the arch is at the end